koku/kafka_utils/utils.py

```diff
@@ -57,7 +58,7 @@ def get_consumer(conf_settings, address=Config.INSIGHTS_KAFKA_ADDRESS):  # pragm
 
 def _get_producer_config(address, conf_settings):  # pragma: no cover
     """Return Kafka Producer config"""
-    producer_conf = {"bootstrap.servers": address, "message.timeout.ms": 1000}
+    producer_conf = {"bootstrap.servers": address, "message.timeout.ms": 3000}
```

Comment:
```suggestion
    producer_conf = {"bootstrap.servers": address}
```
Let's just remove this setting since there is a default value.

Reply:
Removed